python_spike/caseprocessor/pseudorandom/hash_utility.py

```diff
@@ -11,6 +11,4 @@ def generate_hash(string_to_hash: str = None, bytes_to_hash: bytes = None) -> st
 
```

Comment:
I don't think it was ever used in the python spike, it was added when copying the java files as I thought it was used for new case receiver but it wasn't, I can remove it for this prototype, but it might be needed when this comes out of a prototype and we add other services